terasoluna-tourreservation-web/pom.xml

```diff
@@ -46,13 +47,12 @@
         <!-- == Begin TourReservation == -->
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
- [ ] It is wrong that domain depends on env. 
Please see [master/domain](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/master/terasoluna-tourreservation-domain/pom.xml#L18-L19) and [master/web](https://github.com/terasolunaorg/terasoluna-tourreservation/blob/master/terasoluna-tourreservation-web/pom.xml#L49-L53), or [blank-5.0/domain](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/5.0.x/projectName-domain/pom.xml#L23-L24) and [blank-5.0/web](https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/5.0.x/projectName-web/pom.xml#L72-L76).

Reply:
I confirmed the comments above and cherry-picked your commit ( 48b295675cd912250e215693f6000e6aa9afe91d )(and I changed the commit message).